src/database/migrations/20200906143952-create-tourist-center.js

```diff
@@ -0,0 +1,40 @@
+export default {
+  up(queryInterface, Sequelize) {
+    return queryInterface.sequelize
+      .query('CREATE EXTENSION IF NOT EXISTS "uuid-ossp";')
+      .then(() =>
+        queryInterface.createTable("TouristCenters", {
+          countryId: {
+            type: Sequelize.STRING,
+            allowNull: false,
+          },
+          id: {
+            allowNull: false,
+            primaryKey: true,
+            type: Sequelize.UUID,
+            defaultValue: Sequelize.literal("uuid_generate_v4()"),
+          },
+          name: {
+            type: Sequelize.TEXT,
```

Comment:
I think the name, location, and the gallery type should be Sequelize.STRING because their contents will not be as long as the TEXT type.